src/cli/llm/adapter.rs

```diff
@@ -0,0 +1,25 @@
+use std::fmt::{Display, Formatter};
+
+#[derive(Clone)]
+pub enum Adapter {
+    Groq(GroqModel),
+}
+
+#[derive(Clone)]
+pub enum GroqModel {
+    Llama38b8192,
+}
```

Comment:
Add All supported models and adapters that the library support.

Reply:
is there any list for that?